trinity/plugins/builtin/ethstats/ethstats_client.py

```diff
@@ -0,0 +1,126 @@
+import asyncio
+import datetime
+import json
+import platform
+
+import websockets
+
+from trinity.utils.version import construct_trinity_client_identifier
+
+class EthstatsClient:
+    def __init__(self, websocket: websockets.client.WebSocketClientProtocol, node_id: str, server_url: str, server_secret: str) -> None:
+        self.websocket = websocket
+        self.send_queue = asyncio.Queue()
+
+        self.node_id = node_id
+        self.server_url = server_url
+        self.server_secret = server_secret
+
+        self.client = construct_trinity_client_identifier()
+        self.os = platform.system()
+        self.os_v = platform.release()
+
+    async def response_handler(self) -> None:
+        async for message in self.websocket:
+            command, data = self.stat_recv(message)
+            if command == 'node-pong':
+                timestamp = round(datetime.datetime.now().timestamp() * 1000)
```

Comment:
I assume you want `UTC` time which is `datetime.datetime.utcnow()`

Reply:
Fixed to use `utcnow()` (this actually doesn't matter, as it calculates delta between two calls, but it's good idea to use UTC here to avoid potential issues).

Reply:
If all that is needed is a whole number delta in seconds, then this should be updated to use [`time.monotonic()`](https://docs.python.org/3.5/library/time.html#time.monotonic)

Reply:
I think you can ignore my comment here, I mis-interpreted your comment.  `utcnow()` is correct.